tests/test_all.py

```diff
@@ -218,7 +221,7 @@ def test_download_timeout(self):
         self._assert_enabled(self.spider, self.settings, download_timeout=120)
 
     def test_hooks(self):
-        proxyauth = b'Basic Foo'
+        proxyauth = basic_auth_header('foo', '')
```

Comment:
The new implementation requires the header to be actually valid, and `Foo` is not valid base64.